views/pages/HomePage.jsx

```diff
@@ -54,70 +58,68 @@ const HomePage = () => {
       setNotification({ open: true, message: error.message })
     }
   }
-  const sort = async (newSortBy, newSortDirection) => {
-    const participantList = await fetchParticipants({
-      sortBy: newSortBy,
-      sortDirection: newSortDirection,
-      searchSubjects: normalizeSearchSubjects(searchSubjects),
-    })
-
+  const onSort = async (newSortBy, newSortDirection) => {
     setSortBy(newSortBy)
     setDirection(newSortDirection)
-    setParticipants(participantList)
   }
 
-  const handleSearch = async (e) => {
-    setSearchSubjects(e)
-
-    const participantsList = await fetchParticipants({
-      sortBy,
-      sortDirection,
-      searchSubjects: normalizeSearchSubjects(e),
-    })
-
-    setParticipants(participantsList)
-    setRowcount(participantsList.length)
-
-    if (e.length === 0) {
-      const participantsList = await fetchParticipants()
-
-      setParticipants(participantsList)
-    }
+  const handleSearch = async (formData) => {
+    setSearchSubjects(formData.participants)
   }
   const normalizeSearchSubjects = (searchSubjects) =>
     searchSubjects.map(({ value }) => value)
 
   useEffect(() => {
     fetchParticipants().then((participantsList) => {
-      const dropDownOptions = participantsList.map(({ study, subject }) => ({
-        value: `${subject}`,
-        label: `${subject} in ${study}`,
-      }))
+      if (initialLoad) {
+        const dropDownOptions = participantsList.map(({ study, subject }) => ({
+          value: `${subject}`,
+          label: `${subject} in ${study}`,
+        }))
+        setSearchOptions(dropDownOptions)
+        setInitialLoad(false)
+      }
 
       setParticipants(participantsList)
-      setSearchOptions(dropDownOptions)
     })
-  }, [])
+  }, [fetchParticipants, setParticipants, setSearchOptions])
 
   return (
     <>
-      <ReactSelect
-        placeholder="Search a study or participant"
-        value={searchSubjects}
-        onChange={handleSearch}
-        options={searchOptions}
-        autoFocus={true}
-        components={components}
-        isMulti
-      />
-      <ParticipantsTable
-        participants={participants}
-        onUpdate={handleUserUpdate}
-        onSort={sort}
-        sortProperty={sortBy}
-        sortDirection={sortDirection}
-        sortable
-      />
+      <Box sx={{ p: '20px' }}>
```

Comment:
do we need a fragment if the component is wrapped in a Box..?

Reply:
sure don't! https://github.com/AMP-SCZ/dpdash/pull/479/commits/b898e7a9465094559e5e9939062fe5810a2ba309